### DISCUSSION TITLE
Datatables

### DISCUSSION THREAD

**Discussion by reddeerishere** [mdbootstrap/TW-Elements#1516] [Share ideas & request features]
I need the datatables component - an extended table.
Like the one here:
https://mdbootstrap.com/docs/standard/data/datatables/

The most important for me is to have a search bar that will allow me to filter the results in the table.

**Comment by filipkappa** [comment]
Thanks for submitting this idea!

It gathered so much traction that we have added a task for this feature:
https://github.com/mdbootstrap/Tailwind-Elements/issues/1556

You can track our progress live here:
https://tailwind-elements.com/docs/standard/getting-started/changelog/

**Comment by filipkappa** [comment]
We added this to Tailwind Elements! You can now use the feature:
https://tailwind-elements.com/docs/standard/data/datatables/
